README.md

```diff
@@ -49,6 +47,17 @@ To run this sample:
 6. Run the sample:
     - mvn clean compile exec:java
 
+## Use latest Key Vault SDK
+
+The Key Vault secrets SDK here is **com.azure.azure-security-keyvault-secrets**, if you are using the [latest](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets) version of the key vault SDK package, please refer to the following examples:
+
+* [IdentityReadmeSamples.java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/IdentityReadmeSamples.java) shows multiple ways to authenticate the Key Vault client via DefaultAzureCredential, device code, client secret or certificate in addition to others.
+
+* [HelloWorld.java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/HelloWorld.java)  - Examples for common Key Vault tasks:
+
+    * Create a secret inside the Key Vault
+    * Get the secret
+
 ## More information
 
 [http://azure.com/java](http://azure.com/java)
```

Comment:
```suggestion
https://azure.com/java
```

Reply:
Markdown by default understands links, you don't need to specify it. Also use https by default.